snuba/web/query.py

```diff
@@ -164,15 +165,21 @@ def _run_query_pipeline(
     else:
         query_runner = partial(
             _run_and_apply_column_names,
-            timer,
-            query_metadata,
-            request.referrer,
-            robust,
-            concurrent_queries_gauge,
+            timer=timer,
+            query_metadata=query_metadata,
+            attribution_info=request.attribution_info,
+            robust=robust,
+            concurrent_queries_gauge=concurrent_queries_gauge,
         )
 
     record_missing_tenant_ids(request)
+    # NOTE: Something which is not immediately clear from looking at the code is that the
```

Comment:
This should probably be in the doc string.

Reply:
done